plugins/size/src/utils/CalcSizeUtils.ts

```diff
@@ -1,4 +1,5 @@
 import { monorepoName, createLogger } from '@design-systems/cli-utils';
+import { cosmiconfigSync as load } from 'cosmiconfig';
```

Comment:
:bulb: would prefer to use `@design-systems/load-config` here, but trying to link that resulted in a weird circular dependency issue that broke installation (see previous build). Got tired of fighting with that, but if someone else wants to tackle that optimization then by all means.